### PR TITLE
Refactor README.md

### PR DESCRIPTION
**Pull request by dgzlopes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,33 +3,79 @@
...
+    });
 
-## Use case for xk6-disruptor
+    // Check that there is at least one target.
+    const targets = disruptor.targets();
+    if (targets.length != 1) {
+        throw new Error("expected list to have one target");
+    }
 
-The main use case for xk6-disruptor is to test the resiliency of an application of diverse types disruptions by reproducing their effects, but without having to reproduce their root-causes. For example, inject delays in the HTTP requests an application makes to a service  without having to stress or interfere with the infrastructure (network, nodes) on which the service runs, or affecting other workloads in unexpected ways.
```
Comment (pablochacin) [inline README.md:26]: Why are you removing the use case? I think it is important to provide context.

File: README.md [hunk 2]
```diff
@@ -3,33 +3,79 @@
...
+            averageDelay: 500,
+            errorRate: 0.1,
+            errorCode: 500,
+        },
+        30
+    );
+}
+```
 
-In this way, xk6-disruptor make reliability tests repeatable, predictable and limits their blast-radius. These are important characteristic in order to incorporate this kind of tests in the test suits of applications deployed on shared infrastructures such as staging environments.
```
Comment (pablochacin) [inline README.md:28]: I think the concept in this line is relevant, maybe we can rephrase it to make lighter.

File: README.md [hunk 3]
```diff
@@ -3,33 +3,79 @@
...
+## Features
+
+The project at this time is highly focused on providing an excellent experience to test systems running in Kubernetes. Other platforms are not supported at this time.
```
Comment (pablochacin) [inline README.md:73]: It sounds like a piece of marketing to me. I personally don't feel comfortable using this language in the repository of an open source project.

**Review by pablochacin — COMMENTED** [review]
I have some concerns about the tone and removing some content I still consider relevant.

File: README.md (hunk 1 above)
Comment (dgzlopes) [reply README.md:26]: Adding it back! Yup, agree.

File: README.md (hunk 2 above)
Comment (dgzlopes) [reply README.md:28]: Tried to :eyes: 

Any thoughts?

File: README.md (hunk 3 above)
Comment (pablochacin) [reply README.md]: For me the problematic phrase is "excellent experience".

**Comment by pablochacin** [comment]
@dgzlopes I updated the PR to resolve the conflicts with changes I made in other PRs to the README. Please check I did not remvove any of your proposed changes by mistake.

File: README.md (hunk 3 above)
Comment (dgzlopes) [reply README.md]: Yup. I removed it :+1:

File: README.md [hunk 4]
```diff
@@ -3,35 +3,94 @@
...
+            errorRate: 0.1,
+            errorCode: 500,
+        },
+        30
+    );
+}
+```
+
+## Features
+=======
```
Comment (dgzlopes) [inline README.md:72]: Should we remove this? @pablochacin

Comment (pablochacin) [reply README.md:72]: I think so

File: README.md [hunk 5]
```diff
@@ -3,35 +3,94 @@
...
 
-</div>
+## Why another Chaos Engineering tool?
 
-The xk6-disruptor is a [k6](https://k6.io) extension providing fault injection capabilities to test system's reliability under turbulent conditions. Think of it as "like unit testing, but for reliability". 
+Compared to other tools in the space, this one is purposely designed and built to provide the best experience for developers trying to make their systems more reliable:
 
-This project aims to aid developers in building reliable systems, implementing the goals of "Chaos Engineering" discipline in a k6 way - with the best developer experience as its primary objective. 
+- Everything as code.
+  - No need to mess with YAML or learn a new DSL.
```
Comment (pablochacin) [inline README.md:29]: I like the "no need to learn a new DSL" but I don't feel comfortable with "mess with YAML" part. I find it a weak differentiation mostly considering we are in the Kubernetes ecosystem on which yaml is still prevalant. 
```suggestion
  - No need to learn a new DSL.
```

File: README.md [hunk 6]
```diff
@@ -3,35 +3,94 @@
...
+- Fast to adopt with no day-two surprises.
+  - No need to deploy and maintain a fleet of agents or operators.
+- Easy to extend and integrate with other types of tests.
+  - No need to try to glue five tools together to get the job done.
```
Comment (pablochacin) [inline README.md:33]:
```suggestion
  - No need to try to glue multiple tools together to get the job done.
```

File: README.md [hunk 7]
```diff
@@ -3,35 +3,94 @@
...
 
-The xk6-disruptor is a [k6](https://k6.io) extension providing fault injection capabilities to test system's reliability under turbulent conditions. Think of it as "like unit testing, but for reliability". 
+Compared to other tools in the space, this one is purposely designed and built to provide the best experience for developers trying to make their systems more reliable:
 
-This project aims to aid developers in building reliable systems, implementing the goals of "Chaos Engineering" discipline in a k6 way - with the best developer experience as its primary objective. 
+- Everything as code.
+  - No need to mess with YAML or learn a new DSL.
+- Fast to adopt with no day-two surprises.
+  - No need to deploy and maintain a fleet of agents or operators.
+- Easy to extend and integrate with other types of tests.
```
Comment (pablochacin) [inline README.md:32]: I like the concept, not sure it is clear enough. Make the assumption the person who is reading is already familiar with the other types of tests that k6 can do. I would be more explicit.

Comment (dgzlopes) [reply README.md:32]:
```suggestion
- Easy to extend and integrate with other [types of tests](https://k6.io/docs/test-types/introduction/).
```

Yeah, agree. What do you think if we link to the docs?

Comment (pablochacin) [reply README.md:32]: That should work.

File: README.md [hunk 8]
```diff
@@ -3,35 +3,93 @@
...
-In this way, xk6-disruptor make reliability tests repeatable, predictable and limits their blast-radius. These are important characteristic in order to incorporate this kind of tests in the test suits of applications deployed on shared infrastructures such as staging environments.
+## Use cases
+
+The main use case for xk6-disruptor is to test the resiliency of an application of diverse types of disruptions by reproducing their effects without reproducing their root causes. For example, inject delays in the HTTP requests an application makes to a service without having to stress or interfere with the infrastructure (network, nodes) on which the service runs or affect other workloads in unexpected ways.
+
+In this way, xk6-disruptor make reliability tests repeatable and predictable while limiting their blast radius. These are essential characteristics to incorporate these tests in the test suits of applications deployed on shared infrastructures such as staging environments.
 
 ## Learn more
 
 Check the [get started guide](https://k6.io/docs/javascript-api/xk6-disruptor/getstarted) for instructions on how to install and use `xk6-disruptor`.
```
Comment (pablochacin) [inline README.md:86]:
```suggestion
Check the [get started guide](https://k6.io/docs/javascript-api/xk6-disruptor/get-started) for instructions on how to install and use `xk6-disruptor`.
```

File: README.md [hunk 9]
```diff
@@ -3,35 +3,93 @@
...
+<p align="center">  
+</div>
 
-![logo](assets/logo.png)
+<blockquote align="center">
+⚠️ <strong>Important</strong> ⚠️ 
+<br>This project is still in its early stages and under active development.
+<br>The API is subject to change, and there may be bugs.
+<br>Use at your own risk. Thanks!
+</blockquote>
```
Comment (pablochacin) [inline README.md:20]: I don't like how this note looks centered below the logo.  I would move below the next paragraph

```suggestion
```

Comment (pablochacin) [reply README.md:20]: I don't find how to add this block after the next paragraph because there is already another suggestion on that paragraph. I think you can add this block later in another commit:

File: README.md [hunk 10]
```diff
@@ -3,35 +3,93 @@
...
+}
+```
+
+## Features
 
-## Use case for xk6-disruptor
+The project, at this time, is intended to test systems running in Kubernetes. Other platforms are not supported at this time.
 
-The main use case for xk6-disruptor is to test the resiliency of an application of diverse types disruptions by reproducing their effects, but without having to reproduce their root-causes. For example, inject delays in the HTTP requests an application makes to a service  without having to stress or interfere with the infrastructure (network, nodes) on which the service runs, or affecting other workloads in unexpected ways.
+Right now, it offers an [API](https://k6.io/docs/javascript-api/xk6-disruptor/api) for creating disruptors that target one specific type of the component (e.g., Pods) and is capable of injecting different kinds of [faults]([Services](https://k6.io/docs/javascript-api/xk6-disruptor/api/faults), such as errors in HTTP requests served by that component.
```
Comment (pablochacin) [inline README.md:75]:
```suggestion
It offers an [API](https://k6.io/docs/javascript-api/xk6-disruptor/api) for creating disruptors that target one specific type of the component (e.g., Pods) and is capable of injecting different kinds of [faults]([Services](https://k6.io/docs/javascript-api/xk6-disruptor/api/faults), such as errors in HTTP requests served by that component. 
```

**Review by pablochacin — CHANGES_REQUESTED** [review]

File: README.md [hunk 11]
```diff
@@ -3,35 +3,93 @@
...
 
-![logo](assets/logo.png)
+<blockquote align="center">
+⚠️ <strong>Important</strong> ⚠️ 
+<br>This project is still in its early stages and under active development.
+<br>The API is subject to change, and there may be bugs.
+<br>Use at your own risk. Thanks!
+</blockquote>
 
+This extension adds fault injection capabilities to [Grafana k6](https://github.com/grafana/k6). It implements the ideas of the Chaos Engineering discipline and enables Grafana k6 users to test their system's reliability under turbulent conditions.
```
[the remark this answers is not in the document]
Comment (dgzlopes) [reply README.md:22]: I'm sometimes annoyed when the title equals the start of the first sentence.

But yeah, accepting it.

File: README.md [hunk 12]
```diff
@@ -3,35 +3,93 @@
...
+    // Disrupt the targets by injecting HTTP faults into them for 30 seconds
+    disruptor.injectHTTPFaults(
+        {
+            averageDelay: 500,
+            errorRate: 0.1,
+            errorCode: 500,
+        },
+        30
+    );
+}
```
[the remark this answers is not in the document]
Comment (dgzlopes) [reply README.md:71]: I just wanted to compress the script a bit so it doesn't take all the screen state. 

I don't have strong opinions here, either.

File: README.md (hunk 9 above)
Comment (dgzlopes) [reply README.md]: Let me try :+1:

File: README.md [hunk 13]
```diff
@@ -3,35 +3,97 @@
...
+</p>
+<p align="center">  
+</div>
 
-![logo](assets/logo.png)
 
+xk6-disruptor is an extension adds fault injection capabilities to [Grafana k6](https://github.com/grafana/k6). It implements the ideas of the Chaos Engineering discipline and enables Grafana k6 users to test their system's reliability under turbulent conditions.
 
-</div>
+<blockquote align="center">
```
Comment (dgzlopes) [inline README.md:18]: Here? Or did I misunderstand what you meant? :D @pablochacin

**Comment by dgzlopes** [comment]
Sorry for all the commits... I just found there is a way to batch suggestions :( 

We should squash on merge.

File: README.md (hunk 12 above)
[the remark this answers is not in the document]
Comment (pablochacin) [reply README.md:72]: Then I would prefer the way I suggest which is consistent with the examples in the documentation.

File: README.md (hunk 13 above)
Comment (pablochacin) [reply README.md:18]: Yes, only that not centered.

**Comment by pablochacin** [comment]
> We should squash on merge.

Yes, please let's not forget that :pray:

File: README.md (hunk 13 above)
Comment (pablochacin) [inline README.md:18]:
```suggestion
<blockquote >
```

File: README.md [hunk 14]
```diff
@@ -3,35 +3,94 @@
...
-![logo](assets/logo.png)
 
+xk6-disruptor is an extension adds fault injection capabilities to [Grafana k6](https://github.com/grafana/k6). It implements the ideas of the Chaos Engineering discipline and enables Grafana k6 users to test their system's reliability under turbulent conditions.
 
-</div>
+<blockquote>
+⚠️ <strong>Important</strong> ⚠️ 
+<br>This project is still in its early stages and under active development.
+<br>The API is subject to change, and there may be bugs.
+<br>Use at your own risk. Thanks!
```
Comment (pablochacin) [inline README.md:22]:
```suggestion
> ⚠️  xk6-disruptor is in the alpha stage, undergoing active development. We do not guarantee API compatibility between releases - your k6 scripts may need to be updated on each release until this extension reaches v1.0 release.
```

Comment (pablochacin) [reply README.md:22]: @dgzlopes In an early version of the README we decided that "use at your own risk" may deter people form using the tool. We prefer this wording.

Comment (dgzlopes) [reply README.md]: Sounds good!

**Review by pablochacin — APPROVED** [review]
